gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/api/search/SubscriptionCriteria.java

```diff
@@ -58,6 +61,10 @@ public class SubscriptionCriteria {
         this.planSecurityTypes = builder.planSecurityTypes;
     }
 
+    public Collection<String> getIds() {
```

Comment:
in the url you will have `&ids=1&ids=2&ids=...`
is it working well with a collection ? or is it better to have `&=ids1,2,3`, I don't know if we have a strong convention about passing list of values ?

Reply:
We tried with `&=ids1,2,3` and it seems our management API can't handle it. That's why we preferred `&ids=1&ids=2&ids=...`